cves/kernel/CVE-2020-26088.yml

```diff
@@ -188,8 +192,9 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    There is no mention of any specification violation, and I do not believe that there should be a reason to say there was. Checking privileges is not a specified requirement.
```

Comment:
You could argue that this is a violation of the man pages - https://linux.die.net/man/7/capabilities. According to this manpage, CAP_NET_RAW is required to create raw sockets. However, that check was not properly made prior to this vulnerability being patched.

Reply:
Good point! Thanks for that, I overlooked MAN pages. I have added this consideration.